package com.br.desafio.backend.exceptions

class UserNotFound(msg: String) : RuntimeException(msg)